### ISSUE TITLE
How many packages kronos's packages.txt are really necessary?

### ISSUE THREAD

**Issue by derek-schultz** [Locu/chronology#368]
```
libevent-dev
libpcre3-dev
python-all-dev
python-pip
uuid-dev
```

I got Kronos running on OS X without installing any of them.

**Comment by marcua** [comment]
packages.txt is for ubuntu :)

**Comment by derek-schultz** [comment]
Yes, I know that packages.txt is used by `make install` on Ubuntu, but I was wondering if they are still necessary to get it working or if they are an artifact of earlier dependencies. I was surprised to have Kronos run without having to install anything manually.

**Comment by marcua** [comment]
libevent-dev  -- for gevent
libpcre3-dev -- no idea---do we do fast regex anywhere?
python-all-dev -- presumably for python?
python-pip -- useful :)
uuid-dev -- necessary for timeuuid
